common/alert/reports/a11y-1.json

```diff
@@ -0,0 +1,847 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/alert/en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "",
```

Comment:
Add a little text description of the component that is being evaluated, like "Assessment of the instances of the alert component on the referred page."